src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+class Rectangle extends Figure {
```

Comment:
The class name 'Rectangle' should not start with an empty line above its declaration.